content/html/HTML.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "HTML: Hypertext Markup Language"
+mdn_url: https://developer.mozilla.org/en-US/docs/Web/HTML
+recipe: landing-page
+related_content: /content/related_content/html.yaml
+link_lists:
+  - title: Learn HTML
+    pages:
+      - "/content/learn/html/Introduction_to_HTML.md"
```

Comment:
I think I'd prefer either `"./Introduction_to_HTML.md"` or `"/learn/html/Introduction_to_HTML.md.md"`. 
That feels more right and is much more similar to how NodeJS works. That `content` folder is not for human consumption. Similarly, now you never type `const lib = require('node_modules/some-lib')`.

But I don't feel strongly about it. I just prefer more "real" paths.

Reply:
I don't think "./Introduction_to_HTML.md" works because we're not in the same directory. We could do "/learn/html/Introduction_to_HTML.md".

I'd be happy to make this change but I expect it'll go a bit deeper than this because there are probably other places where we are including "content" in the path.

Reply:
Been thinking more about it and I'm feeling more strongly about this one but I think it can wait because it's a bit out of context. A bit like HTML docs refers to images as either 
```html
<img src="../image.png"> or <img src="/images/foo.png">
```
but they don't do...
```html
<img src="/django_root/images/bar.png">
```

Speaking of images, some day we'll have images and then it becomes important that those images actually resolve when viewed in the GitHub UI for markdown preview I think. 

Long story short; some other day.